cmd/database/database_create.go

```diff
@@ -87,45 +88,51 @@ var dbCreateCmd = &cobra.Command{
 
 		dbVersions, err := client.ListDBVersions()
 		if err != nil {
-			utility.Error("Unable to list database versions %s", err)
+			utility.Error("Failed to fetch database versions: %s", err)
 			os.Exit(1)
 		}
 
-		if software == "" {
-			software = "MySQL"
-		}
+		software = strings.ToLower(software)
+		softwareIsValid := false
+		softwareVersionIsValid := false
 
-		if softwareVersion == "" && software == "MySQL" {
-			softwareVersion = "8.0"
+		validSoftwares := map[string][]string{
+			"mysql":      {"mysql"},
+			"postgresql": {"postgresql", "psql"},
 		}
 
-		if software == "PostgreSQL" && softwareVersion == "" {
-			softwareVersion = "14"
+		apiSoftwareNames := map[string]string{
+			"mysql":      "MySQL",
+			"postgresql": "PostgreSQL",
 		}
 
-		softwareIsValid := false
-		softwareVersionIsValid := false
-		if software != "" {
-			for swName, version := range dbVersions {
-				if swName == software {
+		canonicalSoftwareName := ""
+		for swName, aliases := range validSoftwares {
+			for _, alias := range aliases {
+				if alias == software {
 					softwareIsValid = true
-					for i, v := range version {
-						if v.SoftwareVersion == version[i].SoftwareVersion {
+					canonicalSoftwareName = apiSoftwareNames[swName]
+					for _, v := range dbVersions[canonicalSoftwareName] {
+						if v.SoftwareVersion == softwareVersion {
 							softwareVersionIsValid = true
+							break
 						}
 					}
 				}
-
 			}
 		}
 
 		if !softwareIsValid {
-			utility.Error("The provided software name is not valid. Make sure you use correct capitalization (e.g. MySQL, PostgreSQL)")
+			utility.Error("The provided software name is not valid. Make sure you use correct capitalization (eg: MySQL, PostgreSQL)")
```

Comment:
I am confused,  if the change was to allow the usage of lowercase names of the database (ideally case insensitive) why do we still have this error asking for correct capitalization?